pootle/apps/pootle_store/unit/filters.py

```diff
@@ -163,7 +163,7 @@ def get_search_fields(self, sfields):
         for field in sfields:
             if field in self.search_mappings:
                 search_fields.update(self.search_mappings[field])
-            elif field in search_fields:
+            elif field in self.search_fields:
```

Comment:
this is a mixed bag of fixes - im happy to land some - but im expecting all will land.